lgc/patch/Gfx9ConfigBuilder.cpp

```diff
@@ -1013,11 +1018,10 @@ void ConfigBuilder::buildVsRegConfig(ShaderStage shaderStage, T *pConfig) {
     SET_REG_FIELD(&pConfig->vsRegs, PA_CL_VS_OUT_CNTL, VS_OUT_MISC_SIDE_BUS_ENA, true);
 
     if (gfxIp.major == 9) {
-    } else if (gfxIp.major == 10) {
-    } else
-      llvm_unreachable("Not implemented!");
+    } else {
+    }
```

Comment:
This empty else statement looks suspicious. Could we get rid of it or add a comment why we need it?

Reply:
Sure, done